src/store.ts

```diff
@@ -138,8 +170,113 @@ const useStoreImpl = create<IState>((set: SetState<IState>, get: GetState<IState
         return { ...state, finished: 0, start: 0 }
       })
     },
+    addKeyBinding: (action: string, newKey: Key) => {
+      set((state) => {
+        const index = state.keyboardBindings.findIndex(({ action: stateAction }) => action === stateAction)
+
+        const keyboardBindingsCopy = deduplicateKeys(newKey, state.keyboardBindings)
+
+        if (index !== -1) {
```

Comment:
make this:
```suggestion
        if (index < 0) {
```
move to the top and return and then the rest will run if it is a valid index

Reply:
done